config/features.js

```diff
@@ -0,0 +1,30 @@
+const { ENVIRONMENTS } = require('./constants');
```

Comment:
This is a placeholder implementation of the accessor in the [`flag-editor` package](https://github.com/near/near-wallet/pull/1771) which will be imported directly when available. In the interim this implementation is subject to change.